index.html

```diff
@@ -0,0 +1,65 @@
+<!doctype html>
+<html lang="de">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="description" content="Unsere liebsten Mitbewohner auf 4 Pfoten">
+  <title>Die Hauskatze</title>
+  <link rel="stylesheet" href="C:\Users\codersbay\Desktop\Webaufgaben\web-aufgabe-02-kapitel-4-5-BeWie24\stil.css">
+</head>
+<body>
+    <header>
+        <nav>
+            <ul>
+                <li><a href="index.html" class="aktuell">Home</a></li>
+                <li><a href="infos.html">weitere Informationen</a></li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+    <h1>Die allzeit beliebte Hauskatze</h1>
+    <hr>
+    <p> <img src="C:\Users\codersbay\Desktop\Webaufgaben\web-aufgabe-02-kapitel-4-5-BeWie24\bilder\katze.jpg" alt="Katze"></p>
+    <hr>
+    <h2>Infos</h2>
+    <dl>
+        <dt>Ordnung</dt>
+        <dd>Raubtiere (Carnivora)</dd>
+        <dt>Unterfamilie</dt>
+        <dd>Kleinkatzen</dd>
+        <dt>wissenschaftlicher Name</dt>
+        <dd><i>Felis catus</i></dd>
+    </dl>
+  <p><article>Die Merkmale der 

```

Comment:
p steht für paragraph = text
deshalb gehört er in den artikel
er umschließt einen Text